maths/gelu.py

```diff
@@ -0,0 +1,49 @@
+"""
+This script demonstrates the implementation of the Gelu function.
+
+The function takes a vector x of K real numbers as input and
+resukts x * sigmoid(1.702*x).
+The Gaussian Error Linear Unit (GELU), is a high-performing neural
+network activation function.
```

Comment:
```suggestion
The function takes a vector x of K real numbers as input and returns the x * sigmoid(1.702*x).
The Gaussian Error Linear Unit (GELU), is a high-performing neural network activation function.
```

Reply:
I was not able to pass some checks if I kept that sentence in 1 line therefore I have to reformat that and keep in more than 1 line I hope that's okay.

Thanks for the spelling mistake : )